cmd/postcli/main.go

```diff
@@ -275,6 +276,36 @@ func saveKey(key ed25519.PrivateKey) error {
 }
 
 func cmdVerifyPos(opts config.InitOpts, fraction float64, logger *zap.Logger) error {
+	log.Println("cli: verifying key.bin")
+
+	filename := filepath.Join(opts.DataDir, edKeyFileName)
+	data, err := os.ReadFile(filename)
+	if err != nil {
+		return fmt.Errorf("key read from disk error: %w", err)
```

Comment:
ditto:
```suggestion
		return fmt.Errorf("could not read keys from %s: %w", filename, err)
```

... unless `err` already contains the path.